### ISSUE TITLE
GUI installation issues - TypeError in Cpython 1.5.0

### ISSUE THREAD

**Issue by Gypaets** [SteveDoyle2/pyNastran#652]
I'm trying to run the pyNastran GUI.

I've installed `pyNastran` using the command `python3.8 -m pip install --user pyNastran`. When I try to run `pyNastranGUI` on my terminal I get a TypeError.

```
[user@domain ~]$ pyNastranGUI
qt_version = 'pyqt5'
supported_formats = ['nastran', 'nastran', 'abaqus', 'avus', 'bedge', 'surf', 'ugrid', 'ugrid3d', 'cart3d', 'fast', 'lawgs', 'obj', 'openfoam_hex', 'openfoam_shell', 'openfoam_faces', 'panair', 'shabp', 'stl', 'su2', 'tecplot', 'tetgen', 'usm3d', 'avl']
Traceback (most recent call last):
  File "/home/user/.local/bin/pyNastranGUI", line 8, in <module>
    sys.exit(cmd_line())
  File "/home/user/.local/lib/python3.8/site-packages/pyNastran/gui/gui.py", line 111, in cmd_line
    MainWindow(inputs)
  File "/home/user/.local/lib/python3.8/site-packages/pyNastran/gui/main_window.py", line 142, in __init__
    self.start_logging()
  File "/home/user/.local/lib/python3.8/site-packages/pyNastran/gui/gui_common.py", line 1300, in start_logging
    log = SimpleLogger(
  File "/usr/local/lib/python3.8/site-packages/cpylog-1.5.0-py3.8.egg/cpylog/__init__.py", line 113, in __init__
    assert nlevels >= 1, nlevels
TypeError: '>=' not supported between instances of 'function' and 'int'

```
When I install cpython-1.4.0 instead of 1.5.0 the problem disappears.

@SteveDoyle2 If you want I can replace all references to `nlevels` in pyNastran with a different variable name so that there are no name collisions. `pyNastran` would work then with more recent versions of `cpylog`. Otherwise I'd suggest updating the [installation instructions](https://pynastran-git.readthedocs.io/en/1.3/installation/installation.html#installation-from-release#packages) to `cpylog == 1.4.0` instead of `cpylog >= 1.4.0`.

**Comment by SteveDoyle2** [comment]
That’s odd.  Python tracebacks leave a bit to be desired in that only the
first line of the call is shown, but I believe there should be a **kwargs
on the next line (cui_common.py line 1300).  I can tell you’re probably
relatively up to date on pyNastran, so it’s not that.

What’s strange is I’m running cpylog 1.5, but admittedly it’s not from an
install.  I may have missed something, but I doubt it..

.I’ll take a look, but it should work with 1.4 and 1.5.

**Comment by Gypaets** [comment]
The difference between 1.4.0 and 1.5.0 is that they've added a new argument to the __init__ function of `SimpleLogger`. Since the call in `cui_common.py` relies on the order of the arguments, the lambda function was passed as `nlevels` instead of `log_func`:
![grafik](https://user-images.githubusercontent.com/19589322/127167495-85b1bfbc-c633-408e-9733-b3a4b979345f.png)

If I see it right it was already fixed in 609ec9ec76bd287fbfe516467937fcc7dbf9abd8 - when I clone the last version from the repository, the call to `SimpleLogger()` is already made specifying the name of the arguments. I guess that when you upload the next version to Pypi it'll work out of the box with cpylog 1.5.0.

**Comment by SteveDoyle2** [comment]
Ahhh....I missed the pip install pynastran part.  I thought you were installing from source in a way that I know is possible, but I never do.
